nomos-http-api/Cargo.toml

```diff
@@ -27,5 +27,5 @@ axum = "0.6"
 hyper = { version = "0.14", features = ["full"] }
 utoipa-swagger-ui = { version = "4.0", features = ["axum"] }
 reqwest = { version = "0.11", features = ["blocking", "json"] }
-serde = { version = "1", features = ["derive"] }
 serde_json = "1"
+tokio = { version = "1.32", features = ["full"] }
```

Comment:
why's tokio added as a dev dependency here? I don't see this PR adding any test 🤔

Reply:
Ah, I remember why this was added, previously, it is actually a left-over, previously, I tried to use `#[tokio::test]` in test cases.